trie/sync_bloom.go

```diff
@@ -179,6 +179,9 @@ func (b *SyncBloom) Add(hash []byte) {
 // While the bloom is being initialized, any query will return true.
 func (b *SyncBloom) Contains(hash []byte) bool {
 	bloomTestMeter.Mark(1)
+	if b == nil {
+		return true
+	}
 	if atomic.LoadUint32(&b.inited) == 0 {
```

Comment:
If the bloom is not initialized yet, or already closed, the `b.inited` is set to 0. Yes it's correct.